data/hai/hai reveal 4 middle.txt

```diff
@@ -1022,46 +1031,189 @@ mission "Hai Reveal [A09] History of Appeasement"
+			`	The three of you sit down once Teeneep looks ready to talk.`
+				goto questions
+			label friends
+			action
+				set "Hai Reveal [A09] History of Appeasement: label friends"
+			`	Sayari looks at you apologetically. "She's always been quite a spark, but we usually get along very well."`
+			`	Teeneep adds, "Sayari has been my close friend and mentor since I was a head shorter than her."`
+				goto questions
+
+			# Second phase of conversation: find out about the pirates.
+			label deals
+			action
+				set "Hai Reveal [A09] History of Appeasement: label deals"
+			`	Sayari says, "The Elders have been making deals with pirates for a long time, in return for secrecy or ceasing mischef. We give them credits, supplies for their colonies, or weapons. I'm not privy to the details, but I've know for quite a while."`
+			`	An enraged Teeneep counters. "They gave them our weapons so that they could more efficiently go and enslave other people rather than just dealing with them properly they way they should have! One hundred and seventy three reports, Sayari. That's how many times they've done this, or something like it, and those are just the ones that came up in the sweep. There's no telling how many others might be filed improperly, missing key words, or just outside of my search range.`
+			`	"I thought you were a model of morals, Sayari. Now I know you're no better than they are. No better than... than..."`
+			`	Sayari finishes Teeneep's thought. "No better than you? The curse of power is to sacrifice your morals for protection of people under your care. I think you know that well, Teeneep. Perhaps better than I?" Teeneep looks briefly ashamed when Sayari says that.`
+				goto questions
+			label types
+			action
+				set "Hai Reveal [A09] History of Appeasement: label types"
+			`	Teeneep says, "In the cases I found, a third of the deals included weapons: shield regenerators, guns, turrets, and anti-missile systems. Most of the deals included something else too, like money, non-hostile technology, or supplies for anarchist worlds. For the past five years though, nearly every deal had weapons in the trade." She looks irate at Sayari for a second.`
+				goto questions
+
+			# Third phase of the conversation: decide to fix the problem.
+			label fix
+			action
+				set "Hai Reveal [A09] History of Appeasement: label fix"
+			`	Sayari explains. "The grievance is against the Elders. There's a process to seek redress of grievances with the Elders. Teeneep needs to at least attempt to follow that process, and if she does not go through with normal resolution channels, she will be censured from public life. She must confront the Elders directly."`
+			`	Teeneep makes the effort to give a final, defiant snarl of disgust. "She's right," she grates out. "I hate it but she's right. Especially because several of those reports indicate that the Navy knew of Hai on some ships and stood by while they were taken away. Which makes it your business too."`
```

Comment:
```suggestion
			`	Teeneep makes the effort to give a final, defiant snarl of disgust. "She's right," she grates out. "I hate it, but she's right. Especially because several of those reports indicate that the Navy knew of Hai on some ships and stood by while they were taken away. Which makes it your business, too."`
```